app/routes/circulars/circulars.server.ts

```diff
@@ -172,3 +173,19 @@ export async function put(subject: string, body: string, request: Request) {
     submitter: formatAuthor(user),
   })
 }
+
+export async function circularRedirect(query: string | undefined) {
+  if (!query) return
+  const validCircularSearchStyles =
+    /^(?:\s*GCN\s*)?(?:\s*CIRCULAR\s*)?((\s*-?\d+(?:\.\d)?)\s*)$/i
+  const circularID = validCircularSearchStyles.exec(query)?.[1]
+  if (circularID) {
+    const db = await tables()
+    const result = await db.circulars.get({
+      circularId: Number(circularID),
```

Comment:
```suggestion
  const circularId = parseFloat(validCircularSearchStyles.exec(query)?.[1])
  if (!isNaN(circularID)) {
    const db = await tables()
    const result = await db.circulars.get({
      circularId,
```

Reply:
yeah, I'm still getting the error after restarting the typesense server, are you encountering something similar?

Reply:
I see. Although JavaScript permits this, apparently TypeScript makes a principled (and perhaps controversial) decision not to support it. See, for instance, https://github.com/microsoft/TypeScript/issues/4002 and https://github.com/microsoft/TypeScript/issues/15198.

I think that it works if you replace `parseFloat` with `Number`.

Reply:
I was able to avoid the error by changing it to 
``const circularId = parseFloat(validCircularSearchStyles.exec(query)?.[1] || '')``